code/game/objects/items/eightball.dm

```diff
@@ -41,6 +41,13 @@
 		new /obj/item/toy/eightball/haunted(get_turf(src))
 		qdel(src)
 
+/obj/item/toy/eightball/examine(mob/user)
+	..()
+	if(on_cooldown)
```

Comment:
UGH OK I WHEN I CAN UNDERSTAND LADY REMIE AND HER WISDOM I WILL FIX THIS PROPERLY